arch/RISCV/riscv_helpers_ast2str.h

```diff
@@ -0,0 +1,235 @@
+#ifndef __AST2STR_HELPERS_H__
+#define __AST2STR_HELPERS_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "riscv_helpers_rvconf.h"
+
+#define RISCV_TEMP_BUFFER_MAX_LEN 32
+
+#define spc(ps, plen, c)                                                       \
+  *ps = " ";                                                                   \
+  *plen = 1
+
+#define opt_spc spc
+
+#define sep(ps, plen, c)                                                       \
+  *ps = " , ";                                                                 \
+  *plen = 3
+
+static inline void hex_bits(uint64_t bitvec, char **s, size_t *len,
```

Comment:
Add Doxygen comments for these helper functions. Also, maybe move to the capstone utils instead? cc @Rot127

Reply:
Sorry, ignore it. See my comment in the final review message.